Source/MMParser.m

```diff
@@ -710,6 +726,345 @@ - (MMElement *)_parseHorizontalRuleWithScanner:(MMScanner *)scanner
     return element;
 }
 
+
+#pragma mark - Checklist
+
+/**
+ Check the following markers:
+ 
+ - [ ]
+ - [x]
+ - [X]
+ + [ ]
+ * [ ]
+ 1. [ ]
+ 
+ @param scanner MMScanner
+ @return ture if the check list marker exist, false otherwise.
+ */
```

Comment:
This should be formatted to match the existing docstrings in the project.

Also, _true_ has been misspelled.